.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Pull Request Template
+
+## File a Bug
+
+All changes should be associated with a bug. The WebKit project is currently using [Bugzilla](https://bugs.webkit.org) as our bug tracker. Note that multiple changes may be associated with a single bug.
+
+## Provided Tooling
+
+The WebKit Project strongly recomends contributors use [`Tools/Scripts/git-webkit`](https://github.com/WebKit/WebKit/tree/main/Tools/Scripts/git-webkit) to generate pull requests, see [Setup](https://github.com/WebKit/WebKit/wiki/Contributing#setup) and [Contributing Code](https://github.com/WebKit/WebKit/wiki/Contributing#contributing-code) for how to do this.
+
+## Template
+
+If a contributor wishes to file a pull request manually, the template is bellow. Manually filed pull requests should contain their commit message as the pull request description, and their commit message should be formatted like the template bellow.
+
+Additionally, the pull request should be mentioned on [Bugzilla](https://bugs.webkit.org), labels applied to the pull request matching the component and version of the [Bugzilla](https://bugs.webkit.org) associated with the pull request and the pull request assigned to its author.
+
+#### _hash_
```

Comment:
Why is this here? Does it somehow help tooling? It fails whenever we force push, which is going to be very often. E.g. I just did so earlier today in #291, so now there it points to a stale commit instead of what is actually going to be merged.

Reply:
I'll remove this from the template.